src/status_im2/contexts/syncing/events.cljs

```diff
@@ -72,14 +72,31 @@
                          :event    :syncing/input-connection-string-for-bootstrapping}))))]
     (native-module/prepare-dir-and-update-config "" default-node-config-string callback)))
 
+;; TODO :: Password verification is currently based on length but we have to do it realtime
```

Comment:
let's remove dead code. you could add it as a comment in the pr review if it's necessary to communicate this out 👍